tests/recipe_manager_tests.js

```diff
@@ -0,0 +1,96 @@
+const {
+  getRecipes,
+  getRecipe, 
+  createRecipe, 
+  updateRecipe,
+  deleteRecipe
+} = require('../js/recipe_manager');
+
+
+
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
+
+let database = {
+  recipes: [{
+    id: 4,
+    title: "Pasta alla Genovese"
+  },
+  {
+    id: 8,
+    title: "Pizza"
+  }]
+};
+
+
+test('getRecipes returns recipes', () => {
+  window.db = database;
+  expect(getRecipes()).toEqual([{
+    id: 4,
+    title: "Pasta alla Genovese"
+  },
+  {
+    id: 8,
+    title: "Pizza"
+  }]);
+});
+
+test('getRecipe returns recipe by id ', () => {
+  window.db = database;
+  expect(getRecipe(4)).toEqual({
+    id: 4,
+    title: "Pasta alla Genovese"
+  });
+});
+
+test('getRecipe didn\' t returns recipe by not existing id ', () => {
+  window.db = database;
+  expect(getRecipe(10)).toBe(null);
+});
+
+test('createRecipe returns new recipe\'s id', () => {
```

Comment:
I'd mix single/double quotes to improve readability rather than escaping quotes in the string

Reply:
You're right